src/sections/assets/AssetsModal.tsx

```diff
@@ -144,34 +86,39 @@ export const AssetsModalContent = ({
     }
   }
 
-  if (!tokens.length)
-    return (
-      <>
-        <SAssetsModalHeader>
-          <Text
-            color="basic700"
-            fw={500}
-            fs={12}
-            lh={12}
-            tTransform="uppercase"
-          >
-            {t("selectAssets.asset")}
-          </Text>
-          <Text
-            color="basic700"
-            fw={500}
-            fs={12}
-            lh={12}
-            tTransform="uppercase"
-          >
-            {t("selectAssets.your_balance")}
-          </Text>
-        </SAssetsModalHeader>
-        {[1, 2, 3, 4, 5, 6].map((n) => (
-          <AssetsModalRowSkeleton key={n} />
-        ))}
-      </>
-    )
+  const sortedTokens = tokens.allowed.sort((a, b) => {
```

Comment:
This will also sort the original array when used in different parts of the app, because `array.sort` is mutable function, which might not be desirable.

Copy it with `[...tokens.allowed]` or `tokens.allowed.slice(0)` and then sort.

Using `useMemo` hook could also be good here.